protocol-dashboard/src/components/CallToActionBanner/CallToActionBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import cn from 'classnames'
+import { PropsWithChildren, ReactNode } from 'react'
+
+import styles from './CallToActionBanner.module.css'
+
+export type CallToActionBannerProps = PropsWithChildren<{
+  href: string
+  text: ReactNode
+  pillText: ReactNode
+}>
+
+type PillProps = PropsWithChildren<{}>
+const Pill = ({ children }: PillProps) => {
+  return <div className={styles.pill}>{children}</div>
+}
+
+type BannerProps = PropsWithChildren<{}>
+const Banner = ({ children }: BannerProps) => {
```

Comment:
this is cool, though honestly not sure if we need sub-components just for a container?